sktime/forecasting/fbprophet.py

```diff
@@ -162,6 +165,7 @@ def __init__(
         self.freq = freq
         self.add_seasonality = add_seasonality
         self.add_country_holidays = add_country_holidays
+        self.fit_kwargs = fit_kwargs or {}
```

Comment:
it is important that - in adherence with `sklearn` interface specifications, what is written to `self` is always identical with the `__init__` arg.

If you want a case distinction, you could use `self._fit_kwargs`.

Reply:
Good catch! I was not aware of the specification. Please review my changes